### PR TITLE
added shufflenet, fixed profiling issue

### PR DESCRIPTION
**Pull request by AnushreeBannadabhavi**
PR #17 has to be merged before merging changes in this PR.

### CODE REVIEW

File: runtime_lib/ProfilingLib.c [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
 #include "Utils.h"
 
-static long long opcodecount[OPCODE_CYCLE_ARRAY_LEN] = {0};
+static long long unsigned opcodecount[OPCODE_CYCLE_ARRAY_LEN] = {0};
 
 void doProfiling(int opcode) {
   assert(opcodecount[opcode] >= 0 && 
          "dynamic instruction number too large to be handled by llfi");
-  opcodecount[opcode]++;
+  opcodecount[opcode]++;
```
Comment (uditagarwal97) [inline runtime_lib/ProfilingLib.c:13]: Please remove the trailing whitespaces.

File: sample_programs/shufflenetv2_10/image.c [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+extern "C" {
+  OMTensorList *run_main_graph(OMTensorList *);
+}
+
+void export_layer_output_to_json(OMTensorList *, char*, char*);
+
+int main(int argc, char *argv[]) {
+
+// Input pointers needed for the model
```
Comment (uditagarwal97) [inline sample_programs/shufflenetv2_10/image.c:30]: Please add indentation before this comment.

**Review by uditagarwal97 — CHANGES_REQUESTED** [review]

File: runtime_lib/ProfilingLib.c (hunk 1 above)
Comment (AnushreeBannadabhavi) [reply runtime_lib/ProfilingLib.c]: Made changes in the commit: [097935c](https://github.com/DependableSystemsLab/LLTFI/pull/18/commits/097935c54300329580a27712ec07b0d2cdb294a3)

File: sample_programs/shufflenetv2_10/image.c (hunk 2 above)
Comment (AnushreeBannadabhavi) [reply sample_programs/shufflenetv2_10/image.c]: Made changes in the commit: [097935c](https://github.com/DependableSystemsLab/LLTFI/pull/18/commits/097935c54300329580a27712ec07b0d2cdb294a3)

**Pull request merged**